### DISCUSSION TITLE
Action Button: Room cleaning question

### DISCUSSION THREAD

**Discussion by rafinesse** [merdok/homebridge-miot#411] [General]
Hi all, regarding the room cleaning action button:

The {"piid": 1,"value": 18} entry indicated a room clean

what do 1 and 18 stand for and where do I get them from in the Miot spec metadata fetcher?

Thanks!

**Comment by merdok** [comment]
You have this explained in the guide here: https://github.com/merdok/homebridge-miot/blob/main/docs/robotcleaner.md#dreame
Those are hidden fixed values indicating a room clean are always needs to be there.
